cobigen/cobigen-core-parent/cobigen-core/src/main/java/com/devonfw/cobigen/impl/CobiGenFactory.java

```diff
@@ -51,6 +52,20 @@ public static CobiGen create(URI configFileOrFolder) throws InvalidConfiguration
         return createBean;
     }
 
+    /**
+     * Creates a new {@link CobiGen}
+     *
+     * @return a new instance of {@link CobiGen}
+     * @throws IOException
+     *             if the {@link URI} points to a file or folder, which could not be read.
+     * @throws InvalidConfigurationException
+     *             if the context configuration could not be read properly.
+     */
+    public static CobiGen create() throws InvalidConfigurationException, IOException {
+        URI configFileOrFolder = ConfigurationUtil.findTemplatesLocation();
```

Comment:
I think we should provide a proper error message in case no templates / configuration folder could be found as the error message in `public static CobiGen create(URI configFileOrFolder)` is not providing a proper message in case of null parameter. Maybe we can change the error message to some more meaningful which would also give a CLI user proper knowledge of what has happened.